config/src/main/java/com/typesafe/config/impl/DefaultTransformer.java

```diff
@@ -48,7 +48,12 @@ static AbstractConfigValue transform(AbstractConfigValue value,
                 }
                 break;
             case LIST:
-                // can't go STRING to LIST automatically
+                // can't go STRING to LIST automatically unless the string is empty
+                if (s.equals("")) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/hp/config/file/2444096817/issues/source?bid=3245965&fileBranchId=3245971#l52)